manifest.yml

```diff
@@ -0,0 +1,5 @@
+---
+applications:
+- name: compose-broker
+  memory: 128M
+  instances: 2
```

Comment:
This should probably specify the buildpack as well to avoid having to run the auto-detection (which slows deployment down).